config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
@ogra1 if you mean mounting it as part of https://github.com/snapcore/core-build/blob/master/initramfs/scripts/ubuntu-core-rootfs instead of using writable-paths, that is fine for me. Note that there will be a slight difference with grub/uboot as in this case we would be loop-mounting a folder from the system partition instead of a separate boot partition (which we do not have in this case).